static/scripts/loggedin.js

```diff
@@ -250,4 +250,21 @@ $(document).ready(function () {
         const loaderClassList = document.querySelector(".preload-screen").classList;
         loaderClassList.add("hidden");
     });
+    
+    // from: https://stackoverflow.com/a/187557
+    jQuery.expr[":"].Contains = jQuery.expr.createPseudo(function(arg) {
+        return function( elem ) {
+            return jQuery(elem).text().toUpperCase().indexOf(arg.toUpperCase()) >= 0;
+        };
+    });
+    // js course search/filter
+    $("input.js-search").on("keyup", e => {
+        if(e.key === "Escape") $(e.target).val("");
+        if(e.key === "Unidentified") {
+            e.preventDefault();
```

Comment:
use `return false` to do both at once ([article](https://stackoverflow.com/questions/1357118/event-preventdefault-vs-return-false))

`return (e.key === "Unidentified")` at the end